gimme

```diff
@@ -398,7 +398,7 @@ _list_versions() {
 }
 
 _list_known() {
-	local exp="go([[:alnum:]\.]*)\.src.*" # :alnum: catches beta versions too
+	local exp="go([[:alnum:]\\.]*)\\.src.*" # :alnum: catches beta versions too
```

Comment:
side-rant: these are bogus complaints, nothing you can do about it, but backslash inside double quotes is _only_ special when followed by dollar, backtick, double-quotes, backslash itself, or newline.  Otherwise, it's a regular character.

This behavior is explicitly mandated by POSIX.

So this linting tool is making the code uglier and harder to read because it doesn't understand the actual rules and someone has gone on a semi-informed jihad to change things.

Reply:
We can ignore these instead :smiley:  I didn't know that jihad could be used in this context, but I'm pretty sure I don't like it.